user-guide/install.md

```diff
@@ -41,90 +24,152 @@ The Ambassador Edge Stack is deployed to Kubernetes from the command line via ou
+   metadata:
+     name: quote
+     namespace: ambassador
+   spec:
+     ports:
+     - name: http
+       port: 80
+       targetPort: 8080
+     selector:
+       app: quote
+   ---
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: quote
+     namespace: ambassador
+   spec:
+     replicas: 1
+     selector:
+       matchLabels:
+         app: quote
+     strategy:
+       type: RollingUpdate
+     template:
+       metadata:
+         labels:
+           app: quote
+       spec:
+         containers:
+         - name: backend
+           image: quay.io/datawire/quote:0.2.7
+           ports:
+           - name: http
+             containerPort: 8080
+   ```
+
+2. Deploy the `quote` service to the cluster by typing `kubectl apply -f quote.yaml`.
+
+3. Now, create a `Mapping` configuration that tells Ambassador to route all traffic from `/backend/` to the `quote` service. Copy the YAML and save to a file called `quote-backend.yaml`.
+
+   ```
+   ---
+   apiVersion: getambassador.io/v2
+   kind: Mapping
+   metadata:
+     name: quote-backend
+     namespace: ambassador
+   spec:
+     prefix: /backend/
+     service: quote
+   ```
+
+4. Apply the configuration to the cluster by typing `kubectl apply -f quote-backend.yaml`.
+
+5. Test the configuration by typing `curl -k https://<hostname>/backend/` or `curl -k https://<IP address>/backend/`:
+
+   ```
+   (⎈ |gke_rdl-1:default)$ curl -k https://aes.ri.k36.net/backend/
+   {
+    "server": "idle-cranberry-8tbb6iks",
+    "quote": "Non-locality is the driver of truth. By summoning, we vibrate.",
+    "time": "2019-12-11T20:10:16.525471212Z"
+   }
+   ```
+
+## A single source of configuration
+
+1. In the Ambassador Edge Stack, Kubernetes serves as the single source of configuration. Changes made on the command line (via `kubectl`) are reflected in the UI, and vice versa. This enables a consistent configuration workflow. You can see this in action by navigating to the Mappings tab. You'll see an entry for the `quote-backend` Mapping that was just created on the command line.
+
+2. Likewise, if you type `kubectl get hosts`, you'll see the `Host` resource that was created (assuming you configured TLS):
+
+   ```
+   (⎈ |rdl-1:default)$ kubectl get hosts
+   NAME               HOSTNAME           STATE   PHASE COMPLETED   PHASE PENDING   AGE
+   aes.ri.k36.net     aes.ri.k36.net     Ready                                    158m
+   ```
+
+## Developer Onboarding
+
+The Quote service we just deployed publishes its API as a Swagger document. This API is automatically detected by the Ambassador Edge Stack and published.
+
+1. In the Edge Policy Console, navigate to the **APIs** tab. You'll see the documentation there for internal use.
+
+2. Navigate to `https://<hostname>/docs/` or `https://<IP address>/docs/` to see the externally visible Developer Portal. This is a fully customizable portal that you can share with third parties who need to information about your APIs. 
```

Comment:
without the backslash at the end it doesn't work. Maybe emphasize that more.

Reply:
Also maybe it would make sense to add an "API Docs" button into the API page of the UI, instead of forcing the user to take additional action.